docs/roadmap.md

```diff
@@ -70,16 +70,16 @@ or a new tool for the documentation in our roadmap…
 ## Front-Commerce’s roadmap
 
 > **Coming soon**: the first 2 projects using Front-Commerce are deployed in
-> production and there are some new features needed (eg: Wishlist). On the
-> other hand, we are working on the 1.0 release which will means a stable API.
+> production and there are some new features needed (e.g.: Wishlist). On the
+> other hand, we are working on the 1.0 release which will mean a stable API.
 >
 > But the cool thing is that we have a lot of traction these days and our partners have
-> several Front-Commerce projects in the pipe, with decisions in the next few weeks.
+> several Front-Commerce projects in the pipe, with decisions coming in the next few weeks.
 >
 > For all these reasons the Roadmap will evolve a lot by mid-April and we will
 > keep you updated.
 
-If you want to have an imprecise vision of what comes next here is a
+If you want to have an overview of what comes next here is a
```

Comment:
open for discussion, I didn't really like the term "imprecise" but I'm not sure here

Reply:
The thing is that it is less or more a guess… that being said it is very likely to be correct, so let’s use overview :metal: